src/commands.jl

```diff
@@ -300,8 +300,9 @@ end
 
 const always_run_recursive_interpret = Ref(false)
 function any_active_breakpoint()
-    for bp in _breakpoints
```

Comment:
Hmm, how did this just not error...

Reply:
It errors while running Debuggers test suite, so maybe its just not tested in this repo.